gnomad_qc/v5/data_ingestion/create_vds.py

```diff
@@ -16,8 +16,11 @@
 def main(args):
     """Create VDS from text file of VDSes or GVCFs paths and write to output path."""
     hl.init(
-        log="/tmp/gvcf_combiner.log", default_reference="GRCh38", tmp_dir=args.temp_path
+        log="/tmp/gvcf_combiner.log",
+        tmp_dir=args.temp_path,
+        copy_spark_log_on_error=True,
     )
+    hl.default_reference("GRCh38")
```

Comment:
just wondering- why move default_reference out of the init?

Reply:
I got a message the old init default reference was being deprecated so just got ahed of that here.